### PR TITLE
feat: Migrating Models & Constants

### PR DESCRIPTION
**Pull request by michaelmyers**
The first two packages open sourced.

This is a straight copy-paste from the closed source versions except for this [commit](https://github.com/stentorium/stentor/pull/1/commits/b77f934205a864cf2f009dfd5ac7281add671aff), which accounts for the updates required for the latest TypeScript version.

### CODE REVIEW

File: packages/stentor-constants/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [1.14.1-alpha.6](https://github.com/stentorium/stentor/compare/stentor-constants@1.14.1-alpha.5...stentor-constants@1.14.1-alpha.6) (2020-01-03)
```
Comment (chrsdietz) [inline packages/stentor-constants/CHANGELOG.md]: Do we care about any of this for an initial public release?

File: packages/stentor-models/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@xapp/config/tsconfig.json",
```
Comment (chrsdietz) [inline packages/stentor-models/tsconfig.json]: Remove anything from "@xapp"?

File: packages/stentor-constants/CHANGELOG.md (hunk 1 above)
Comment (michaelmyers) [reply packages/stentor-constants/CHANGELOG.md]: Not really but they exist https://www.npmjs.com/package/stentor-models so we should keep the record of them

File: packages/stentor-models/tsconfig.json (hunk 2 above)
Comment (michaelmyers) [reply packages/stentor-models/tsconfig.json]: Some things will stay with @xapp prefix, like this shared config repository.

**Pull request merged**